build.gradle

```diff
@@ -23,6 +23,50 @@ buildscript {
 
 plugins {
     id("io.gitlab.arturbosch.detekt").version("1.0.0-RC16")
+    id("com.nimbledroid.profiler").version("1.2.1")
+}
+
+// Placeholder for ND plugin configuration.
+//
+// The ND plugin expects this to be where gradle
+// stores configuration information. Variables
+// are set by the setNimbledroidConfiguration task.
+nimbledroid {
+}
+
+// Set the appropriate variables to perform the ND upload.
+//
+// Use the API key from the environment. Always upload a
+// nightly build.
+task setNimbledroidConfiguration {
+    doLast {
+        if (System.env.ND_APIKEY == null) {
+                throw new Exception("API key must be set in the environment (ND_APIKEY) for nimbledroidUpload")
+        }
+        nimbledroid.apiKey = "${System.env.ND_APIKEY}";
+        nimbledroid.apkFilename = "geckoNightly/fenixNightly/app-geckoNightly-armeabi-v7a-fenixNightly-unsigned.apk"
```

Comment:
I'm surprised this doesn't need the prefix, `app/build/outputs/apk/`

Reply:
The plugin prepends some part of the location, per the documentation.